### ISSUE TITLE
the diff run fails with a missing file

### ISSUE THREAD

**Issue by mackell** [stdevel/satprep#44]
When I run the satprep_diff.py I see:
```
satprep_diff.py errata-snapshot-report-xxxxxxxxxxx-20150802-*csv
INFO:satprep_diff:Assuming file1 ('errata-snapshot-report-xxxxxxxxx-20150802-0059.csv') is the first snapshot.
INFO:satprep_diff:Snapshot and monitoring checkboxes won't be pre-selected as we don't have a valid .vlog!
Traceback (most recent call last):
  File "./satprep_diff.py", line 492, in <module>
    main(options)
  File "./satprep_diff.py", line 196, in main
    f_log = open(options.verificationLog, 'r')
IOError: [Errno 2] No such file or directory: ''
```

I also attempted using -x and both of the csv files, same result.

Running under RHEL 6.6 x86_64 Linux.

**Comment by mackell** [comment]
I added the touch statements:
+ touch errata-snapshot-report-imcltspacew-20151001-0059.vlog
+ touch errata-snapshot-report-imcltspacew-20151001-2359.vlog
+ ./satprep_diff.py -x /satreport/SatPrep/errata-snapshot-report-imcltspacew-20151001-0059.csv -x /satreport/SatPrep/errata-snapshot-report-imcltspacew-20151001-2359.csv -i /satreport/SatPrep/DukeEnergy.jpg -f 'Duke-Energy maintenance report'
INFO:satprep_diff:Assuming file2 ('/satreport/SatPrep/errata-snapshot-report-imcltspacew-20151001-2359.csv') is the first snapshot.
INFO:satprep_diff:Snapshot and monitoring checkboxes won't be pre-selected as we don't have a valid .vlog!
Traceback (most recent call last):
  File "./satprep_diff.py", line 492, in <module>
    main(options)
  File "./satprep_diff.py", line 462, in main
    letter.write(s.substitute(titleHostname=host, ip=this_ip, date=this_date, owner=this_owner, systemStandalone=this_standalone, systemCluster=this_cluster, hintsClusterTest=hintsClTest, hwCheckNo=this_hwCheckNo, hwCheckNotes=this_hwCheckNotes, vmSnapYes=this_vmSnapYes, vmSnapNo=this_vmSnapNo, vmSnapNotes=this_vmSnapNotes, rebootNo=this_NoReboot, rebootNotes=this_RebootNotes, errata=this_errataTable, orientation=options.pageOrientation+",", footer=options.footer, logo=options.logoImage, monSchedYes=this_monYes, monSchedNo=this_monNo, monSchedNotes=this_monNotes, BackupNo=this_backupNo, BackupNoNotes=this_backupNoNotes, AntivirNo=this_antivirNo, AntivirNoNotes=this_antivirNoNotes))
UnboundLocalError: local variable 'this_vmSnapNo' referenced before assignment

**Comment by stdevel** [comment]
Fixed in commit 9e478d522f94d39fb002a2f031868735b51fe5ad
Is this also working for you?

**Comment by stdevel** [comment]
Hey mackell,
any news on this?

Best regards,
Christian.
